api/views/yangSearch/response_row.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, elastic_hit: dict) -> None:
         self.origin = self._set_origin(elastic_hit['organization'])
         self.organization = elastic_hit['organization']
         self.description = elastic_hit['description']
-        self.maturity = elastic_hit.get('maturity-level', '')
+        self.maturity = 'ratified' if elastic_hit.get('rfc') else ''
```

Comment:
Maybe it would be enough to use only: 
`self.rfc = elastic_hit.get('rfc')`
and work only with this True/False value
Or does we need this value explicitly somewhere?

Reply:
This is the output row, so we show it in the response